mflowgen/glb_tile/custom-init/outputs/floorplan.tcl

```diff
@@ -52,4 +52,53 @@ setFlipping s
 
 planDesign
 
+proc snap_to_grid {input granularity} {
+   set new_value [expr ceil($input / $granularity) * $granularity]
+   return $new_value
+}
+
+set horiz_pitch [dbGet top.fPlan.coreSite.size_x]
+set srams [get_cells *sram_array*]
+set sram_width [dbGet [dbGet -p top.insts.name *sram_array* -i 0].cell.size_x]
+set sram_height [dbGet [dbGet -p top.insts.name *sram_array* -i 0].cell.size_y]
+set sram_start_y [expr [dbGet top.fPlan.box_lly] + 50]
+set sram_start_x [expr [dbGet top.fPlan.box_llx] + 50]
+set sram_spacing_y 0
+set sram_spacing_x_even 0
+# Magic number
+set sram_spacing_x_odd 5
+set bank_height 8
+
+set y_loc $sram_start_y
+set x_loc $sram_start_x
+set col 0
+set row 0
+foreach_in_collection sram $srams {
+  set sram_name [get_property $sram full_name]
+  # Does this line make sense?
+  set y_loc [snap_to_grid $y_loc $horiz_pitch]
+  if {[expr $col % 2] == 0} {
+    placeInstance $sram_name $x_loc $y_loc MY -fixed
+  } else {
+    placeInstance $sram_name $x_loc $y_loc -fixed
+  }
```

Comment:
To get rid of some DRCs, you may want to add some pgnet routing blockages around the SRAMs, like I did for the memory tile. See this commit: https://github.com/StanfordAHA/garnet/commit/44e49758f9497b6f215eb65a50aafd1f13c7b485